From c60ced33593dcf29e19c782905587a440329b0cf Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Fri, 23 Oct 2015 11:29:42 +0200
Subject: [PATCH] nm: install the .name file into /usr/lib/NetworkManager/VPN

It's a preferred location for system-provided plugins. We keep the old location
for compatibility.
---
 Makefile.am | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 2864eb2..a7a30e2 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -5,9 +5,12 @@ SUBDIRS = properties auth-dialog po
 dbusservicedir = $(sysconfdir)/dbus-1/system.d
 dbusservice_DATA = nm-strongswan-service.conf
 
-nmvpnservicedir = $(sysconfdir)/NetworkManager/VPN
+nmvpnservicedir = $(prefix)/lib/NetworkManager/VPN
 nmvpnservice_DATA = nm-strongswan-service.name
 
+oldnmvpnservicedir = $(sysconfdir)/NetworkManager/VPN
+oldnmvpnservice_DATA = $(nmvpnservice_DATA)
+
 @INTLTOOL_DESKTOP_RULE@
 
 nm-strongswan-service.name: $(srcdir)/nm-strongswan-service.name.in
-- 
2.4.3

